userdocs/src/usage/eks-private-cluster.md

```diff
@@ -60,6 +60,21 @@ privateCluster:
 
 The endpoints supported in `additionalEndpointServices` are `autoscaling`, `cloudformation` and `logs`.
 
+### Skipping endpoint creations
+
+If a VPC has already been created with the necessary AWS endpoints set up and linked to the subnets described in the EKS documentation,
+`eksctl` can skip creating them by providing the option `skipEndpointCreation` like this:
+
+```yaml
+privateCluster:
+  enabled: true
+  skipEndpointCreation: true
```

Comment:
is having it under `privateCluster` the appropriate place? Would anybody ever want to use this in other non-private scenarios? Just wondering. I'm not super context-heavy on this so I might be missing the obvious :smile:

Reply:
VPCEndpoints are only relevant in private clusters.

Reply:
cool!